internxt/drive-server-wip
# Fix/unit tests isolated

This pull request adds an github action only for UNIT TESTS while also fixing them so we can start adding them slowly.

How to test build for production:

1. Clone your .env.development and change its name to .env (to simulate production .env file)
2. Run `docker run --rm -it $(docker build -q .)` in the root folder of the project.
3. Database connection is going to fail but the container should start without any extra error.
